src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_adexchangebuyer.v1.4.json.baseline

```diff
@@ -4,24 +4,24 @@
 // 1. If not already done, enable the Ad Exchange Buyer API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/adexchangebuyer
-// 2. This sample uses Application Default Credentials for authentication.
-//    If not already done, install the gcloud CLI from
-//    https://cloud.google.com/sdk/ and run
-//    'gcloud beta auth application-default login'
-// 3. Install the Node.js client library and Application Default Credentials
+// 2. Install the Node.js client library and Application Default Credentials
```

Comment:
Do remove the ADC reference. If you could run the compilechecks to make sure we're not generating errors, that would be great. I am working on generalizing/automating the compile check, but it will be a little bit yet before that's ready.